assets/js/components/ExecutionResults/ExecutionResults.jsx

```diff
@@ -101,9 +101,11 @@ function ExecutionResults({
   executionData,
   executionError,
   clusterSelectedChecks = [],
+  savedFilters,
```

Comment:
Could we maybe add an empty list as default?
To make things a bit more secure, and in fact it would even help to story book

Reply:
Let me check if nothing explodes, sure

Reply:
I managed to assign a default prop but I had to declare it outside the inline scope. Left a comment about that, hope it's ok